src/main/xerus/monstercat/tabs/TabPlaylist.kt

```diff
@@ -0,0 +1,56 @@
+package xerus.monstercat.tabs
+
+import javafx.beans.property.SimpleStringProperty
+import javafx.beans.value.ObservableValue
+import javafx.scene.control.SelectionMode
+import javafx.scene.control.TableColumn
+import javafx.scene.control.TableView
+import javafx.scene.input.MouseButton
+import javafx.util.Callback
+import xerus.ktutil.javafx.fill
+import xerus.monstercat.api.Player
+import xerus.monstercat.api.Playlist
+import xerus.monstercat.api.response.Track
+
+
+class TabPlaylist : VTab() {
+	var table = TableView<Track>()
+
+	init {
+		prefWidth = 600.0
```

Comment:
you don't need to set the prefWidth, it's already set gobally

Reply:
Fixed